pyls/plugins/jedi_completion.py

```diff
@@ -138,9 +140,9 @@ def _format_completion(d, include_params=True):
         path = path.replace('/', '\\/')
         completion['insertText'] = path
 
-    if (include_params and hasattr(d, 'params') and d.params and
-            not is_exception_class(d.name)):
-        positional_args = [param for param in d.params
+    sig = d.get_signatures()
+    if (include_params and sig and not is_exception_class(d.name)):
+        positional_args = [param for param in sig[0].params
```

Comment:
is `sig`guaranteed to have at least the [0] item?

Reply:
Empty `sig`s don't fullfill the if condition `if ... and sig:`. If you want it more explicit, change it to `... and len(sig) > 0`.

Reply:
Right!, I missed that :-)